server/enterprise/metrics/metrics.go

```diff
@@ -1187,6 +1190,39 @@ func New(ps *platform.PlatformService, driver, dataSource string) *MetricsInterf
 	)
 	m.Registry.MustRegister(m.ClientTimeToFirstByte)
 
+	m.ClientTimeToLastByte = prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Namespace: MetricsNamespace,
+			Subsystem: MetricsSubsystemClientsWeb,
+			Name:      "time_to_last_byte",
+			Help:      "Duration from when a browser starts to request a page from a server until when it receives the last byte of the resource or immediately before the transport connection is closed, whichever comes first. (seconds)",
```

Comment:
Is `a page` always going to be loading the core webapp, or do we ever track other page requests?

Reply:
This is actually for any page, doesn't matter.

Reply:
Does it really matter for a SPA like ours? The initial JS payload is the same for all these cases if I am not wrong. And we are just tracking till `splash_screen_end` which should be the same irrespective of any page IIUC.

Reply:
>  The initial JS payload is the same for all these cases if I am not wrong.

Yeah, this is partly the assumption I'm looking to double check: if the weight of various entry paths matter at present. Agree that it would be ideal if these were all uniform.